app/views/drivers/index.html.slim

```diff
@@ -0,0 +1,24 @@
+table class="table"
+  thread
+    tr
+      th scope="col" Name
+      th scope="col" License Number
+      th scope="col" Mobile Number
+      th scope="col" Email Address
+      th scope="col" Age
+      th scope="col" Notes
+      th scope="col" Status
+      th scope="col" Actions
+  tbody
+    - @drivers.each do |driver|
+      tr
```

Comment:
Also, you can try to extract driver view to the partial
https://guides.rubyonrails.org/action_view_overview.html#partials